tests/test_addons.py

```diff
@@ -886,9 +864,9 @@ def test_delete_action_for_folder_deletes_subfolders_and_creates_trashed_file_no
             }
         }
         views.addon_delete_file_node(self=None, node=self.project, user=self.user, event_type='file_removed', payload=payload)
-        assert_false(StoredFileNode.load(file_node._id))
+        assert_false(GithubFileNode.load(file_node._id))
```

Comment:
Just as an addition test for `StoredFileNode.load` not returning Trashed things can this be reverted back to `StoredFileNode.load`?